### DISCUSSION TITLE
Massive spam posts in Federated Timeline from *.activitypub-troll.cf

### DISCUSSION THREAD

**Discussion by 5long** [mastodon/mastodon#21986] [General]
<img width="326" alt="image" src="https://user-images.githubusercontent.com/159226/205437088-493c432c-8bc9-4f1c-a84b-0e928c284ad9.png">

All the spam posts are from accounts of `*.activitypub-troll.cf`. Although I think those "accounts" are fake and they are not even running "real" instances -- they are running just enough code to spam.

I opened a randomly chosen spam site https://21eaon3p3.activitypub-troll.cf/ directly in browser and this is the code shown in the browser: https://gist.github.com/5long/866eafeae55c24fe36406f91bdb1afe0

Is there anything a user / admin could do to stop the spam?

**Comment by bearice** [comment]
refs: #21977
